notebooks/post-startup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Default post startup script for AI notebooks.
+# AI Notebook post startup scrips are run only when the instance is first created.
+
```

Comment:
Recommend 

```
set -o errexit
set -o nounset
set -o pipefail
```

As they help catch coding or execution errors closer to the point of failure.